config/vector.yaml

```diff
@@ -100,6 +115,7 @@ sinks:
     type: elasticsearch
     inputs:
       - parsed_hs_server_logs
+      - local_file_logs   
```

Comment:
this won't be needed since the file logs are already present in `parsed_hs_server_logs`